packages/zoe/test/unitTests/contracts/test-coveredCall.js

```diff
@@ -100,36 +122,39 @@ test('zoe - coveredCall', async t => {
         t.is(
           invitationValue.installation,
           installation,
-          'installation is atomicSwap',
+          'installation is coveredCall',
         );
         t.is(invitationValue.description, 'exerciseOption');
 
         t.deepEqual(
-          invitationValue.underlyingAsset,
-          moola(3),
-          `underlying asset is 3 moola`,
+          invitationValue.underlyingAssets,
+          { Moola: moola(3) },
+          `underlying assets are 3 moola`,
         );
         t.deepEqual(
           invitationValue.strikePrice,
-          simoleans(7),
-          `strike price is 7 simoleans, so bob must give that`,
+          { Simoleans: simoleans(7), Bucks: bucks(2) },
```

Comment:
I'm a little worried that the additional flexibility at least about the StrikePrice will make coveredCall less recognizable as an option. Would we be better off having them be more narrowly structure to require a specific currency for the strikePrice.

Reply:
I think if we don't have this flexibility, we can't use the coveredCall as reusable "lego" as easily.

Reply:
I think a specific keyword would be good. I'm not sure requiring the currency to be named when starting the contract is an improvement.

Reply:
>  I'm not sure requiring the currency to be named when starting the contract is an improvement.

This is not true. Nothing needs to be named for anyone else. All keywords are independent of each other.